lib/grape-swagger/doc_methods.rb

```diff
@@ -92,11 +124,13 @@ def parse_params(params, path, method)
                         end
         end
         name          = (value.is_a?(Hash) && value[:full_name]) || param
+        description = translate(description, scope,
+                                i18n_keys.map { |key| :"#{key}.params.#{name}" })
 
         parsed_params = {
           paramType:     param_type,
           name:          name,
-          description:   as_markdown(description),
+          description:   as_markdown(description.presence),
```

Comment:
I always found this weird. Ultimately it's just a `nil` or the description, right? Why does it matter here?